napytau/gui/components/checkbox_panel.py

```diff
@@ -75,7 +75,7 @@ def update_data_checkboxes_calculation(self) -> None:
 
         header_label = customtkinter.CTkLabel(
             self.frame_datapoint_checkboxes,
-            text="Datapoints for tau calculation",
+            text="tau calculation",
```

Comment:
I don't have any stakes in this, but @julius-kress should give his opinion. Anyhow, the first word should be capitalized.

```suggestion
            text="Tau calculation",
```

Reply:
I noticed this too late when committing. I was playing around with different looks and forgot to return the label to its previous message.